Week3/homework/ToDoApp_dump.sql

```diff
@@ -0,0 +1,113 @@
+-- MySQL dump 10.13  Distrib 8.0.15, for pc-linux-gnu (i686)
+--
+-- Host: localhost    Database: ToDoApp
+-- ------------------------------------------------------
+-- Server version	8.0.15
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+ SET NAMES utf8mb4 ;
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
+/*!40103 SET TIME_ZONE='+00:00' */;
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+DROP SCHEMA IF EXIST ToDoApp;
+CREATE SCHEMA ToDoApp;
+USE ToDoApp;
+SET AUTOCOMMIT=0;
+
+--
+-- Table structure for table `todolists`
+--
+
+DROP TABLE IF EXISTS `todolists`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+ SET character_set_client = utf8mb4 ;
+CREATE TABLE `todolists` (
+  `User_Id` varchar(50) NOT NULL,
```

Comment:
Why did you choose `varchar(50)`? In my experience, I see mostly `int` or `uuid`

Reply:
Ah, I see, you use `nanoid()`